Projects/Features/Scene/GroupDetailScene/EventDetail/EventDetailView.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  EventDetailView.swift
+//  GroupDetail
+//
+//  Created by 최혜린 on 7/6/24.
+//  Copyright © 2024 com.mashup.gabbangzip. All rights reserved.
+//
+
+import ComposableArchitecture
+import DesignSystem
+import Models
+import SwiftUI
+
+struct EventDetailView: View {
+  private let store: StoreOf<EventDetailCore>
+  private let background = DesignSystem.Colors.gray100
+
+  init(store: StoreOf<EventDetailCore>) {
+    self.store = store
+  }
+
+  var body: some View {
+    ZStack {
+      background
+        .ignoresSafeArea()
```

Comment:
P4:
```suggestion
      DesignSystem.Colors.gray100
        .ignoresSafeArea()
```
Background color을 Suggestion처럼 직접 사용하지 않고 변수로 선언하여 사용하신 이유가 궁금합니다

Reply:
@zekexros background라는 변수로 선언해서 어떤 역할을 하는지 조금 더 명확하게 보여주고 싶었습니다! suggestion 주신 것 처럼 작성해도 괜찮을 것 같네욥